apprentice-react-o-shop/src/utils/history.js

```diff
@@ -2,4 +2,4 @@ import { createBrowserHistory } from 'history';
 
 const history = createBrowserHistory()
 
```

Comment:
In `apprentice-react-o-shop/apprentice-react-o-shop/public/manifest.json` can you remove the icons array. It's throwing an error whenever your route changes.